media-tv/xmltv/xmltv-0.6.1.ebuild

```diff
@@ -0,0 +1,301 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit perl-module
+
+DESCRIPTION="Set of utilities to manage TV listings stored in the XMLTV format"
+HOMEPAGE="http://xmltv.org"
+SRC_URI="https://github.com/XMLTV/xmltv/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+#KEYWORDS="~amd64 ~arm ~ppc ~x86 ~x86-linux"
+### Temporary until I figure out how to get it fixed!
```

Comment:
This is not needed. Just open a re-keywording bug for this package, then open keywording bugs to the dependencies that miss correct KEYWORDS and make it block re-keywording bug for xmltv.

Reply:
Created bugs 719140, 719138

Reply:
Commenting out the whole KEYWORDS entry is also overly aggressive. 

Just remove the ones that are failing, and then later open a keyreq for reinstating them.

( Recommend working out how to use `ekeyword` for this, then its just `ekeyword ^arm ^ppc xmltv-0.6.1.ebuild` )

Reply:
Missing from the quote is the next line:
`KEYWORDS="~amd64 ~x86 ~x86-linux"`
I was unaware of `ekeyword`.